src/firmware/lib/adb/src/adb.cpp

```diff
@@ -491,9 +491,12 @@ void AdbInterface::ProcessCommand(int16_t cmd)
           kbdreg0 = (kbdprev0 << 8) | 0xFF;
 
           // Save timestamp
-          kbskiptimer = millis();
+// @DEBUG switch back after getting cmake to compile QuokkADB
+//          kbskiptimer = millis();
+            kbskiptimer = (time_us_64() / 1000);
```

Comment:
@morio - did you intend for this change to remain? The comment looks like it might have been a temporary debug thing.

Reply:
No, I'll fix it and do some more testing then commit to this PR.